app/src/createExcelFiles/ExcelUtils.js

```diff
@@ -0,0 +1,21 @@
+import * as XLSX from "xlsx";
+
+export function toExcelArrayOfArray(searchItems) {
```

Comment:
The name here is very confusing when you are map/changing data from one schema to another try and name start your functions with map. So in this case it would be something like `mapToExcelArray`